packages/extensions/cloud/n8n-extension.manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://raw.githubusercontent.com/n8n-io/n8n/refs/heads/master/packages/%40n8n/extension-sdk/schema.json",
+	"name": "cloud",
+	"publisher": "n8n",
```

Comment:
Do we need these for an internal module? This won't be published or?

Reply:
Not needed, but we should keep it for now, just in case anyone from the outside starts fiddling with modules, unofficially.